docs/Contributing.asciidoc

```diff
@@ -606,92 +606,6 @@ To keep the test database running after executing tests with the `Makefile`, add
 
 To watch the execution of the UI tests, set the environment variable `NOT_HEADLESS`.
 
-=== Run tests within a container
```

Comment:
That's quite a big section you remove so I think there is some value in it which I think we should keep. Can we instead slim it down and keep something? Just because nobody in our team uses those tests that might still provide an easy entry point for some, isn't it?

Reply:
Yes, I also wasn't sure if I should delete so much. This can be probably kept, because it is not directly tied to the old removed container.